src/core/src/service_interfaces/LifecycleManagerAzure.py

```diff
@@ -96,6 +96,8 @@ def execution_start_check(self):
 
             # Signalling take-over of core state by auto-assessment after safety checks for any competing process
             self.update_core_sequence(completed=False)
+            # Refresh status file in memory to be up-to-date
+            self.status_handler.load_status_file_components()
```

Comment:
1. Test this change on a machine with Auto Patching and Auto Assessment
2. We don't have any tests mocking Auto assessment behavior, to test this scenario. See if it's possible to add that

Reply:
1. Tested on a machine with those attributes, works as intended.
2. Looking into it.

Reply:
When testing on a machine, check/follow these:
1. Create a VM with Auto Patching and Auto Assessment i.e. patchmode = AutomaticByPlatform and assessmentmode = AutomaticByPlatform
2. See if a ConfigurePatching call is triggered. If not, manually invoke ConfigurePatching API
3. Keep on monitoring the API result. Query ContextActivity and check ConfigurePatchingSummary, PatchAssessmentSummary substatus.
4. If both have succeeded, then the changes are valid. If not, the issue still exists

Reply:
Add a task in our backlog for AutoAssessment tests